pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.black]
```

Comment:
Can we remove other config when we add this one?

Reply:
We can remove the `BLACK_PARAMS = --line-length 99` in the Makefile, but as far as I can see nothing else.